examples/multibot.py

```diff
@@ -4,6 +4,7 @@
 from typing import Any, Dict, Union
 
 from aiohttp import web
+from finite_state_machine import form_router
```

Comment:
Looks like it's conflicts with command `make reformat` from contribution guide

Reply:
Nothing had changed after running `make reformat`

Reply:
Please update and merge `aiogram:dev-3.x` to this branch.
We're just updated dependencies of the format checkers and linters.

Reply:
Done